firefox-ios/Client/Frontend/Toolbar+URLBar/TabToolbar.swift

```diff
@@ -102,8 +103,6 @@ class TabToolbar: UIView, SearchBarLocationProvider {
 
 // MARK: - TabToolbarProtocol
 extension TabToolbar: TabToolbarProtocol {
-    var homeButton: ToolbarButton { multiStateButton }
```

Comment:
Having `homeButton` tied to `multiStateButton` cause the fire button to navigate to home page when we only want to perform deletion. I didn't see any use case for why we wanted this code base, let me know if I am missing something! 

It seems `homeButton` is only useful for iPad and the `multiStateButton` handles home state on iPhone.